enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -771,7 +771,33 @@ To answer this question, we need to do the following:
 * Verify that OpenShift Service Mesh will include support for this feature in time for this EP.
 * Evaluate any potential security concerns around this feature.
 
-**Resolution**: TBD.
+**Resolution**: The answer depends on what version of OSSM is selected for dev
+preview. If we use OSSM 2.3, we will **NOT** enable ReferenceGrants. If we use OSSM 2.4,
+we will enable ReferenceGrants. This is because ReferenceGrants are non-functional in
+OSSM 2.3, but functional in OSSM 2.4.
+
+OSSM 2.3 uses Istio 1.14 and Istio 1.14 doesn't support ReferenceGrants; therefore,
+in OSSM 2.3, ReferenceGrants are **non-functional**. This means that, by default,
+Gateway API objects in OSSM 2.3 can reference objects across namespace boundaries,
+such as an HTTPRoute referencing a service in another namespace. A ReferenceGrant
```

Comment:
I would s/This means that/However/.  The fact that the ReferenceGrant API isn't implemented in Istio 1.14 means that it _should prohibit_ cross-namespace references between an HTTPRoute and a service.  It's a bug (and a major one at that, IMO) that Istio 1.14 _allows_ these references.  

It doesn't really matter, though, if we go with OSSM 2.4.

Reply:
Yea I see what you mean, `however` highlights a contrast or contradiction. I'll update.